src/module.c

```diff
@@ -7810,6 +7810,10 @@ size_t RM_GetClusterSize(void) {
 int RM_GetClusterNodeInfo(RedisModuleCtx *ctx, const char *id, char *ip, char *master_id, int *port, int *flags) {
     UNUSED(ctx);
 
+    if (verifyClusterNodeName(id, strlen(ip)) != C_OK) {
+        return REDISMODULE_ERR;
```

Comment:
Let's also update the documentation that the id might be invalid.
```suggestion
    if (verifyClusterNodeName(id, strlen(id)) != C_OK) {
        return REDISMODULE_ERR;
```

Reply:
@madolson Thank you for your comments and corrections. I'll revise it later～